RHEL6_7/system/perl/check

```diff
@@ -6,9 +6,26 @@
 #END GENERATED SECTION
 
 FOREIGN_FOUND=0
-function log_file() {
-        printf '%s\n' "$@" >> "$SOLUTION_FILE" || exit_error
-        FOREIGN_FOUND=1
+log_file() {
+    printf '%s\n' "$@" >> "$SOLUTION_FILE" || exit_error
+    FOREIGN_FOUND=1
+}
+is_module_rh() {
+if [ $# -ne 1 ]; then
+     return 1
+fi
+RPM_NAME=$(rpm -qf --qf '%{NAME}\n' "$1")
+echo "$RPM_NAME" | grep "is not" >> /dev/null
```

Comment:
this is redudant line. The grep is not needed to check whether rpm is installed or not because the rpm utility already returns non-zero value when RPM is not installed. remove the line